openedx/core/storage.py

```diff
@@ -4,9 +4,11 @@
 from django.contrib.staticfiles.storage import StaticFilesStorage, CachedFilesMixin
 from pipeline.storage import PipelineMixin, NonPackagingMixin
 from require.storage import OptimizedFilesMixin
+from openedx.core.djangoapps.theming.storage import ComprehensiveThemingAwareMixin
 
 
 class ProductionStorage(
+        ComprehensiveThemingAwareMixin,
```

Comment:
For the record (mostly for my benefit), this was how it was first implemented:

https://github.com/edx/edx-platform/pull/10090

and this is the PR where it was removed: 

https://github.com/edx/edx-platform/pull/11319